bench/hamming/temp-rearrangement.fpcore

```diff
@@ -1,23 +1,120 @@
 ; -*- mode: scheme -*-
 
-(FPCore (x)
```

Comment:
Still think we should move this file to its own PR

Reply:
Moving it out